benchmark/common_utils.hpp

```diff
@@ -5,15 +5,91 @@
 #include <benchmark/benchmark.h>
 #include <chrono>
 #include <climits>
+#include <fstream>
+#include <functional>
 #include <memory>
+#include <sstream>
+#include <string>
+#include <vector>
 
 #include "blas_meta.h"
+#include "cli_args.hpp"
 
 using index_t = int;
+using Blas3Param = std::tuple<std::string, std::string, int, int, int>;
```

Comment:
this should be named according to the conventions that we have - i.e. `blas_3_param_t`

Reply:
Done